src/main/java/net/imagej/display/DefaultImageDisplayService.java

```diff
@@ -107,45 +107,6 @@ public ImageDisplay getActiveImageDisplay() {
 		return displayService.getActiveDisplay(ImageDisplay.class);
 	}
 
-	@Override
-	public Dataset getActiveDataset() {
-		return getActiveDataset(getActiveImageDisplay());
-	}
-
-	@Override
-	public DatasetView getActiveDatasetView() {
-		return getActiveDatasetView(getActiveImageDisplay());
-	}
-	
-	@Override
-	public Position getActivePosition() {
-		return getActivePosition(getActiveImageDisplay());
-	}
-
-	@Override
-	public Dataset getActiveDataset(final ImageDisplay display) {
-		final DatasetView activeDatasetView = getActiveDatasetView(display);
-		return activeDatasetView == null ? null : activeDatasetView.getData();
-	}
-
-	@Override
-	public DatasetView getActiveDatasetView(final ImageDisplay display) {
-		if (display == null) return null;
-		final DataView activeView = display.getActiveView();
-		if (activeView instanceof DatasetView) {
-			return (DatasetView) activeView;
-		}
-		return null;
-	}
-	
-	@Override
-	public Position getActivePosition(final ImageDisplay display) {
-		if (display == null) return null;
-		final DatasetView activeDatasetView = this.getActiveDatasetView(display);
```

Comment:
@ctrueden is the `this` usage okay?

Reply:
Yeah, you don't even need the `this.` prefix, it's redundant. You can just write `getActiveDatasetView(display)`.

As for whether it will cause problems with the `LegacyImageDisplayService` delegation: I think it will be fine. But we'll see!